db/dcrsqlite/apisource.go

```diff
@@ -1262,3 +1262,17 @@ func (db *wiredDB) CountUnconfirmedTransactions(address string, maxUnconfirmedPo
 	}
 	return
 }
+func (db *wiredDB) GetNumberOfTicketsPerBlock(idx int) (numberOfTicketsPerBlock int64, err error) {
+	data := db.GetBlockVerbose(idx, true)
+	if data == nil {
+		err = errors.New("Unable to get block for block index " + string(idx))
+		return
+	}
+	for _, tx := range data.RawSTx {
```

Comment:
Please refactor this loop into `txhelpers` package, but I think something like this exists already